templates/init-migrate-hook.yaml

```diff
@@ -53,16 +37,46 @@ spec:
       labels:
         {{- include "quorumkeymanager.labels" . | nindent 8 }}
     spec:
+      securityContext:
+        fsGroup: 1001
       restartPolicy: Never
       {{- if .Values.imageCredentials }}
       imagePullSecrets:
         - name: {{ include "quorumkeymanager.imagePullSecretName" . }}
       {{- end }}
+      {{- if .Values.postgresql.tls.enabled }}
+      initContainers:
+        - name: init-chmod-data
+          image: docker.io/bitnami/bitnami-shell:10-debian-10-r185
+          imagePullPolicy: "Always"
```

Comment:
I dont think we need to always pull for new images for this

Reply:
Yes right, I will turn it to `IfNotPresent`